### [Princess Margaret Cancer Centre, University Health Network](https://www.uhn.ca/PrincessMargaret) (UHN)

Princess Margaret Cancer Centre used three (6) panels to sequence samples - UHN-48-V1, UHN-50-V2, UHN-54-V1, UHN-555-V1, UHN-555-V2 and UHN-OCAv3. Each panel is described below:

Illumina TruSeq Amplicon panel (UHN-48-V1): Princess Margaret Cancer Centre used the TruSeq Amplicon Cancer Panel (TSACP, Illumina) to detect single nucleotide variants and small indels from matched tumor-normal sequencing data. Specimens are reviewed by a pathologist to ensure tumor cellularity of at least 20%. Tumors are sequenced to an average unique depth of coverage of approximately 500x and normal blood samples to 100x. Data was processed using one of two workflows:

-   Data analysis of tumor-normal pairs processed by UHN\\\_TSACP\\\_workflow\\\_v2: MiSeq fastq were aligned using (MiSeq Reporter v2.4.60 and the corresponding default version of hg19) followed by local realignment and BQSR using GATK v3.3.0. Somatic sequence mutations were called, using MuTect (v1.1.5) for SNVs and Varscan (v2.3.8) for indels, using both normal and tumor data. Data were filtered to ensure there are no variants included with frequency of 3% or more in the normal sample. Results were filtered to keep only those with tumor variant allele frequency of at least 10%.

-   Data analysis of tumor only processed by UHN_TSACP_tumorONLY_v2_workflow: MiSeq fastq were aligned using (MiSeqReporter v2.4.60 and the corresponding default version of hg19) followed by local realignment and BQSR using GATK v3.3.0. Sequence mutations (SNV and indel) were called using Varscan (v2.3.8). Results were filtered to keep only those with tumor variant allele frequency of at least 10%.

ThermoFisher Ion AmpliSeq Cancer Panel (UHN-50-V2): Princess Margaret Cancer Centre also used the TruSeq Amplicon Cancer Panel (TSACP, Illumina) to detect single nucleotide variants and small indels from matched tumor-normal sequencing data. Specimens were reviewed by a pathologist to ensure tumor cellularity of at least 20%. Tumors were sequenced to an average unique depth of coverage of approximately 500x and normal blood samples to 100x. Ion Torrent data was converted to fastq and sequences were aligned using NextGENe Software v2.3.1. NextGENe Software v2.3.1 provides a version of hg19 (Human_v37_3\_dbsnp_135_dna). NextGENe was used to call SNV and indels. Results were then filtered to keep all with VAF of at least 10% and total coverage of at least 100x.

Illumina TruSeq Myeloid Sequencing Panel (UHN-54-V1): Princess Margaret Cancer Centre also used the TruSeq Myeloid Sequencing Panel (Illumina) to detect single nucleotide variants and small indels in DNA from bone marrow or peripheral blood samples from patients with acute leukemia, myelodysplastic syndrome, or myeloproliferative neoplasms. The diagnosis of each patient was confirmed by hematopathologist using the 2016 revision of the World Health Organization classification system for myeloid neoplasms. Tumors were sequenced to an average unique depth of coverage of approximately 500x. MiSeq fastq were aligned using (MiSeq Reporter v2.4.60 and the corresponding default version of hg19). MiSeq Reporter was then used to call variants. In the "Illumina Experiment Manager", "TruSeq Amplicon Workflow --specific settings" were adjusted as follows: "Export to gVCF --MaxIndelSize" from default "25" to "55". Results were then filtered to keep only those with tumor variant allele frequency of at least 10% and a depth of coverage greater than 500x.

UHN Custom 555 Gene Panel (UHN-555-V1 and UHN-55-V2): Princess Margaret Cancer Centre used a custom Sure Select (Agilent) 555 gene panel to detect single nucleotide variants and small indels in DNA from tumour tissue. Specimens are reviewed by a pathologist to ensure tumor cellularity of at least 10% (usually \>20%). Tumors are sequenced to an average unique depth of coverage of approximately 500x. NextSeq fastq were aligned using bwa-mem and the BAMs were processed using GATK best practices. Variants were called using VarScan v2.3.8. VCF files were filtered to remove known artifacts. Variants that are not on a manually curated list of hotspots that have depth less than 50, or if depth 50 to 100 with frequency \<10%, or if depth was at least 100 with frequency \<5%, were removed.

Ion Oncomine Comprehensive Assay v3 (UHN-OCAv3): Princess Margaret Cancer Centre used the Thermofisher Oncomine Comprehensive Assay v3 to detect relevant single nucleotide variants, small indels, copy number changes and gene fusions. Genomic DNA and RNA were co-isolated and then sequencing was performed on the Ion S5 XL System. Variant calls were generated using Torrent Browser 5.12 (ThermoFisher) with alignment to genome build GRCh37/hg19. Filtering of reportable variants uses the Oncomine Variant Annotator plugin to annotate variants in Ion Reporter 5.12, with variant criteria and thresholds as supplied. The reportable range is 5%-100% variant allele frequency. Minimum acceptable coverage for all reported genomic regions is \>200x. Copy number gains and losses are assessed using the Oncomine Variant Annotator plugin, with thresholds of \<0.5 or \>4 copies reported. Gene fusions are assessed using the Oncomine Variant Annotator plugin, with variant criteria and thresholds as supplied (quality threshold of \>500,000 reads reported as 'pass' for gene fusion sequencing). The assay detects recurrent and novel gene partners with expressed gene fusions of the genes listed and minimum acceptable read count for all reported fusions is \>2000.
