### [Yale University, Yale Cancer Center](https://www.yale.edu/) (YALE)

GENIE samples submitted by Yale belong to one of three targeted NGS panels (1) YALE-HSM-V1, (2) YALE-OCP-V2, or (3) YALE-OCP-V3. The first panel corresponds to the Thermofisher Ion AmpliSeq Cancer Hotspot Panel v2, which is designed to assess hotspot variants in 50 of the most frequently mutated genes in cancer, and is performed as a tumor-only analysis. The latter two panels refer to v2 and v3C of the Thermofisher Oncomine Comprehensive Assay, which provides a more comprehensive assessment of somatic alterations including single nucleotide variants, insertions, deletions, copy number alterations (CNAs), and gene fusions across 143 and 161 genes, respectively. Target region design (i.e. full exonic, hotspot only, intronic, promoter) varies based on known relevance of each gene. Pathologist inspection of an H&E section ensured adequate tumor cellularity (approximately 10% or greater). Tumor samples are enriched for malignant cells by manual microdissection of unstained formalin-fixed, paraffin-embedded (FFPE) tissue sections. If available, germline control DNA from the same patient is obtained either from FFPE non-tumor tissue, from the patient's blood, or from a buccal swab. Subsequent libraries are barcoded and sequenced on either an Ion Torrent PGM™ or an Ion S5™ XL next generation sequencer.

Pre-processing and alignment of reads is performed within Torrent Suite, with TMAP serving as the alignment algorithm. Resulting BAM files are uploaded to the Ion Reporter software for variant detection, as well as CNA and gene fusion assessment for Oncomine samples. The bioinformatics pipeline also uses MuTect2 (GATK) and Strelka (Illumina) to assess somatic variants. Variants are initially filtered based on quality metrics; a minimum read depth of 20x and avariant allelic fraction (VAF) of 0.02 is required. All variants passing quality filters are passed through the Ensembl Variant Effect Predictor for variant annotation. Variants that are intronic or synonymous are filtered at this stage; all other variants are manually reviewed for accuracy before submission to the attending pathologist. Variants below a VAF below 0.05 are not typically reviewed unless tumor cellularity estimates are low. CNA assessment is performed using the IonReporter CNV algorithm,as well as an internally developed workflow that uses the DNAcopy R package. Custom visualizations for amplified genes are used to confirm accuracy of CNAs reported by the pipeline. Only CNAs with a ploidy of 5 or higher are reported. Gene fusion assessment is handled by a custom workflow in the Ion Reporter software which aligns cDNA reads to known fusion breakpoints. A fusion read is mapped successfully if there is an overlap of 70% and exact matches of 66.66%.
